sublime_linter.py

```diff
@@ -241,6 +241,10 @@ def highlight(self, view, errors, hit_time):
         if hit_time and persist.last_hit_times.get(vid, 0) > hit_time:
             return
 
+        bid = view.buffer_id()
+        persist.raw_errors[bid] = errors
+
+        # For compatibility we store the errors SL3 style as well.
```

Comment:
Compatibility with what exactly?

Reply:
We have the new style, flat errors on `raw_errors` which have the type `[buffer_id, [errors]]`. On `persist.errors` we store `[view_id, [line, [error_type, [errors]]`. 

This is merely a safe refactoring strategy. We can slowly change the consumers to use `raw_errors`, and if nobody uses `errors` anymore, we flip names.

Reply:
Should we have todo to do this refactoring?

Reply:
Dunno. I actually like `TODO` and comments in the source. It's open source so a reader sees that there is something that can be done.  

I actually wanted to commit more (welcoming) comments about fishy code, bc there are parts in this software ... really, really hard to read, really not welcoming to do a PR and fix a problem.

Reply:
The code sometimes says: Please, go away, leave me alone.

Reply:
Sure, “leave me alone” is perfectly ok to have in a comment. But if it’s something that really should be be done, ie. it’s technical debt, I want to have a single list of issues that I can manage. Todo comments are not in that list, and have a lower likelihood of getting done.

Reply:
No, I meant, we have a lot of actual code that says "Leave me alone", not comments. 😄 

Anyway, IMO this is a good comment. You to decide if it should go away.

Reply:
oh, aha, yes we have that too 😄 

I'm very much 🤷‍♂️ right now, so yeah. I'll just merge this. :)